pypeit/utils.py

```diff
@@ -11,23 +9,36 @@
 from scipy import interpolate
 
 from astropy import units
-from astropy.io import fits
-from astropy.convolution import convolve, Gaussian1DKernel
 from matplotlib import pyplot as plt
 
 # Imports for fast_running_median
 from collections import deque
 from itertools import islice
 from bisect import insort, bisect_left
+from pypeit.core import pydl
+from pypeit import msgs
 
-#from pydl.pydlutils import math
-#from pydl.pydlutils import bspline
-
-
+def wavegrid(wave_min, wave_max, dwave, osamp=1.0):
```

Comment:
seems like this ought to go in `core/wavecal/`

Reply:
Yes, and utils should go in core, and many of the utils should probably be in other modules as well. Let's discuss code organization in a separate PR.